File: UVA/p12250.py
cases = 1
while True:
    inp = input()    
    if inp == '#':
        break
    else:
        if inp == 'HELLO':
            print('Case {}: ENGLISH'.format(cases))
        elif inp == 'HOLA':
            print('Case {}: SPANISH'.format(cases))
        elif inp == 'HALLO':
            print('Case {}: GERMAN'.format(cases))
        elif inp == 'BONJOUR':
            print('Case {}: FRENCH'.format(cases))
        elif inp == 'CIAO':
            print('Case {}: ITALIAN'.format(cases))
        elif inp == 'ZDRAVSTVUJTE':
            print('Case {}: RUSSIAN'.format(cases))
        else:
            print('Case {}: UNKNOWN'.format(cases))
    cases += 1

File: UVA/p12250.java

import java.util.Scanner;

/**
 * AC, easy, ad hoc, 29 August 2018.
 * @author Faiz Ikhwan
 */
public class p12250 
{
    public static void main(String[] args)
    {
        Scanner input = new Scanner(System.in);
        int counter = 1;
        while(true)
        {
            String inp = input.next();
            
            if(inp.equals("#"))
                break;
            else
            {
                if(inp.equals("HELLO"))
                    System.out.println("Case "+(counter)+": ENGLISH");
                else if(inp.equals("HOLA"))
                    System.out.println("Case "+(counter)+": SPANISH");
                else if(inp.equals("HALLO"))
                    System.out.println("Case "+(counter)+": GERMAN");
                else if(inp.equals("BONJOUR"))
                    System.out.println("Case "+(counter)+": FRENCH");
                else if(inp.equals("CIAO"))
                    System.out.println("Case "+(counter)+": ITALIAN");
                else if(inp.equals("ZDRAVSTVUJTE"))
                    System.out.println("Case "+(counter)+": RUSSIAN");
                else
                    System.out.println("Case "+(counter)+": UNKNOWN");                
                
                counter++;
            }
        }
    }
}
